assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,29 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+() => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+(param) => param;
```

Comment:
single parameters can lose the parentheses (given that you aren't doing anything fancy):
```javascript
const func = param => param;
```